### PR TITLE
Add Hungarian Counties select widget

### PR DESCRIPTION
**Pull request by rlaszlo**

### CODE REVIEW

**Comment by benkonrath** [comment]
Thanks, this looks great. I'll merge this tomorrow at the DjangoCon EU sprints and you can hit the gong!

**Comment by benkonrath** [comment]
@rlaszlo Just one small request that I forgot about. Can you add an entry to the changelog about this new localflavor? Thanks!

File: localflavor/hu/forms.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""HU-specific Form helpers"""
+
+from django.forms.fields import Select
+
+from .hu_counties import HU_COUNTY_CHOICES
+
+
+class HUCountySelect(Select):
+    """A Select widget that uses a list of Hungarian Counties as its choices."""
```
Comment (benkonrath) [inline localflavor/hu/forms.py]: Can you also add a .. versionadded:: 1.3` to this comment. Thanks.

**Comment by rlaszlo** [comment]
Hi @benkonrath ! The pull request is now in the changelog.rst and I put the version in the forms too. Thanks!

**Comment by benkonrath** [comment]
Fixed via b0b36416f9ef1915f0833373edef11db10cc7a77.

**Pull request closed without merging**